builder-buildpackless.toml

```diff
@@ -0,0 +1,10 @@
+description = "Ubuntu 24.04 Noble Numbat base image with no buildpacks included. To use, specify buildpacks at build time."
+
+[lifecycle]
+  version = "0.20.7"
+
+[stack]
+  build-image = "docker.io/paketobuildpacks/ubuntu-noble-build-base:0.0.2"
+  id = "io.buildpacks.stacks.noble"
+  run-image = "index.docker.io/paketobuildpacks/ubuntu-noble-run-base:latest"
```

Comment:
@pacostas Shouldn't the default run image be called `ubuntu-noble-run` instead?

Reply:
correct

Reply:
@c0d1ngm0nk3y I think we are good now with the imges